cookbooks/bcpc/recipes/zabbix-mail.rb

```diff
@@ -0,0 +1,2 @@
+# zabbix uses sendemail
+package 'sendemail'
```

Comment:
Should not this configure `node['bcpc']['zabbix']['scripts']['mail'] = "/usr/local/bin/zbx_mail.sh"`?